main.c

```diff
@@ -28,14 +28,14 @@
 #include "slideextract.h"
 #include <stdio.h>
 #include <stdlib.h>
-#include <unistd.h>
+#include <cargs.h>
 #include <stdbool.h>
 
 //! Print version and exit program
 static void
 version ()
 {
-	puts("slideextract " VERSION "\n\
+	puts("slideextract 0.1.0 \n\
```

Comment:
I found manually setting of this number simplifies the code. We can make it a compile-time value if that is needed.